static/js/components/LearningResourceCard_test.js

```diff
@@ -242,11 +225,70 @@ describe("LearningResourceCard", () => {
       )
       assert.equal(wrapper.find(".drag-handle").exists(), reordering)
       assert.equal(wrapper.find("Subtitle").exists(), !reordering)
-      assert.equal(
-        wrapper.find(".availability-price-favorite").exists(),
-        !reordering
-      )
+      assert.equal(wrapper.find(".start-date-favorite").exists(), !reordering)
       assert.equal(wrapper.find(".cover-image").exists(), !reordering)
     })
   })
+
+  //
+  it(`should render a certificate icon and start date if there is a certificate`, async () => {
+    const object = makeLearningResource(LR_TYPE_COURSE)
+    object.certification = [CERTIFICATE]
+    const { wrapper } = await render({ object })
+    const image = wrapper.find(".audience-certificates").find("img")
+    const startDate = wrapper
+      .find(".start-date")
+      .text()
+      .replace("calendar_today", "")
+    assert.equal(image.prop("src"), CERTIFICATE_ICON)
+    assert.equal(
+      startDate,
+      moment(bestRun(object.runs).best_start_date).format(DISPLAY_DATE_FORMAT)
+    )
+  })
+
+  //
+  it(`should not render a certificate icon or start date if there is no certificate`, async () => {
+    const object = makeLearningResource(LR_TYPE_COURSE)
+    object.certification = []
+    const { wrapper } = await render({ object })
+    assert.isNotOk(
+      wrapper
+        .find(".audience-certificates")
+        .find("img")
+        .exists()
+    )
+    assert.isNotOk(wrapper.find(".start-date").exists())
+  })
+
+  //
+  it(`should render a open content icon if the resource is open content`, async () => {
+    const object = makeLearningResource(LR_TYPE_COURSE)
+    object.audience = [OPEN_CONTENT]
+    const { wrapper } = await render({ object })
+    const image = wrapper.find(".audience-certificates").find("img")
+    assert.equal(image.prop("src"), OPEN_CONTENT_ICON)
+  })
+
+  //
+  it(`should render a professional content icon if the resource is professional content`, async () => {
+    const object = makeLearningResource(LR_TYPE_COURSE)
+    object.audience = [PROFESSIONAL]
+    const { wrapper } = await render({ object })
+    const image = wrapper.find(".audience-certificates").find("img")
+    assert.equal(image.prop("src"), PROFESSIONAL_ICON)
+  })
+
+  //
+  it(`should not render an icon if the resource is neither open content or professional`, async () => {
+    const object = makeLearningResource(LR_TYPE_COURSE)
+    object.audience = []
+    const { wrapper } = await render({ object })
+    assert.isNotOk(
+      wrapper
+        .find(".audience-certificates")
+        .find("img")
+        .exists()
+    )
+  })
```

Comment:
do you think these tests could maybe be parameterized and re-written that way? if not no biggie, I was just thinking they look pretty similar